chain/network/src/types.rs

```diff
@@ -782,6 +782,37 @@ pub struct NetworkConfig {
     pub most_weighted_peer_horizon: u128,
     /// Period between pushing network info to client
     pub push_info_period: Duration,
+    /// Peers on blacklist by IP:Port.
+    /// Nodes will not accept or try to establish connection to such peers.
+    pub blacklist: Vec<PatternAddr>,
+}
+
+/// Used to match a socket addr by IP:Port or only by IP
```

Comment:
Why do we care about port? Do you think if there are multiple instances running on the same ip address we should only blacklist some of them?

Reply:
Because the entire point of blacklist from the get go was to enable tests, and in tests the IP addresses are the same :)

Reply:
I see